tools/topology/topology1/sof/pipe-volume-playback.m4

```diff
@@ -75,14 +75,15 @@ P_GRAPH(pipe-volume-playback, PIPELINE_ID,
 indir(`define', concat(`PIPELINE_SOURCE_', PIPELINE_ID), N_BUFFER(1))
 indir(`define', concat(`PIPELINE_PCM_', PIPELINE_ID), Passthrough Playback PCM_ID)
 
-ifdef(`CHANNELS_MIN',`define(TCHANNELS_MIN, `CHANNELS_MIN')',
-`define(TCHANNELS_MIN, `2')')
+ifdef(`CHANNELS_MIN',`define(`_CHANNELS_MIN', `CHANNELS_MIN')',
```

Comment:
Looks like this has already been fixed in this file, but it's missing the undef. Can we rename ```_CHANNELS_MIN``` as  ```LOCAL_CHANNELS_MIN``` as this is easier to follow.
This is not a problem for topology 2.0

Reply:
Yes, here it was already fixed. But indeed renaming TCHANNELS_MIN to _CHANNELS_MIN makes sense.